beacon-chain/operations/attestations/kv/aggregated.go

```diff
@@ -19,18 +22,23 @@ func (c *AttCaches) AggregateUnaggregatedAttestations() error {
 	if err != nil {
 		return err
 	}
-	return c.aggregateUnaggregatedAttestations(unaggregatedAtts)
+	return c.aggregateUnaggregatedAttestations(context.Background(), unaggregatedAtts)
```

Comment:
This will make tracking these spans impossible in the full context of the operation. Please use a parent context of some kind.